servers/zms/conf/zms.properties

```diff
@@ -577,3 +577,10 @@ athenz.zms.no_auth_uri_list=/zms/v1/schema
 # of results returned to the specified value. The default value is 100. This prevents
 # the server from returning a large number of results when the search criteria is too broad.
 #athenz.zms.search_service_limit=100
+
+# This property specifies the maximum expiry duration in days for user/service/group.
+# The value must be an integer, and the default value is 0.
+# If set to 0, it indicates that there is no expiry limit.
+#athenz.zms.default_user_expiry_days=0
```

Comment:
let's rename these to be athenz.zms.max_default_... (add the max_ component). default value indicates that we'll set this if no value is specified but here we're doing 2 things: first if no value is set, we default to this value and second, if a value is specified then we do not allow the value to be bigger than this (which is max part and not necessarily default).